deeplake/auto/structured/dataframe.py

```diff
@@ -20,6 +30,105 @@ def __init__(self, source):
         if not isinstance(self.source, pd.DataFrame):
             raise Exception("Source is not a pandas dataframe object.")
 
+        self.creds = creds
+        self.creds_key = creds_key
+        self._initialize_params(column_params)
+
+    def _sanitize_tensor(self, input: str):
```

Comment:
@ProgerDav Can you help me by moving this function in a general utils that it can be used wherever a tensor name needs to be sanitized